.development/i18n-uuid/.env.template

```diff
@@ -1 +1,12 @@
 I18NUUID_DEFAULT_LOCALE=nl
+
+I18NUUID_ROOT=todo-path-to-directory-to-check-for-translations
```

Comment:
Ik had deze specifiek verwijderd omdat deze nooit anders is dan de directory boven de development map + en omdat dit de setup moeilijker maakt.

In de env files mogen enkel de secrets staan, al de rest houden we best in Github en fixed